npm/pkg/controlplane/controllers/v2/networkPolicyController.go

```diff
@@ -269,6 +280,15 @@ func (c *NetworkPolicyController) syncAddAndUpdateNetPol(netPolObj *networkingv1
 		return errNetPolTranslationFailure
 	}
 
+	_, policyExisted := c.rawNpSpecMap[netpolKey]
+	var mode metrics.ApplyMode
+	if policyExisted {
+		mode = metrics.UpdateMode
+	} else {
+		mode = metrics.CreateMode
+	}
```

Comment:
```suggestion
var mode metrics.ApplyMode
if _, ok := c.rawNpSpecMap[netpolKey]; ok {
    mode = metrics.UpdateMode
} else {
    mode = metrics.CreateMode
}
```

Reply:
ah I see - you need to use it in the follow up code. Nevermind - you can ignore this suggestion